hyperplane/item.py

```diff
@@ -120,14 +120,17 @@ def __file_thumb_done(self, failed: bool) -> None:
         self.icon.set_visible(failed)
         self.picture.set_visible(not failed)
 
+        self.icon.set_css_classes(["large-icons"])
```

Comment:
These should probably be done in `unbind()`. `bind()` should only run again after `unbind()` has been run.

Reply:
Ah, right. Didn't notice there was a method for this :)